framework/machine-learning/inc/MLTensor.h

```diff
@@ -0,0 +1,149 @@
+/* Copyright (c) 2017 SKKU ESLAB, and contributors. All rights reserved.
+ *
+ * Contributor: Gyeonghwan Hong<redcarrottt@gmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ML_TENSOR_H__
+#define __ML_TENSOR_H__
+
+#include <iostream>
+#include <string>
+#include <vector>
+#include <map>
+
+namespace MLDataType {
+  enum Value {
+    Char = 1,
+    Int32 = 2,
+    Float = 3,
+    Double = 4
+  };
+
+  inline int size(MLDataType::Value dataType) {
+    switch(dataType) {
+      case MLDataType::Char:
+        return 1;
+        break;
+      case MLDataType::Int32:
+        return 4;
+        break;
+      case MLDataType::Float:
+        return 4;
+        break;
+      case MLDataType::Double:
+        return 8;
+        break;
+      default:
+        return 0;
+        break;
+    }
+  }
+};
+
+class MLTensorLayout {
```

Comment:
@sinban04 
```MLTensorLayout``` is also defined in ```MLTensor.h```.

Reply:
oh i'm sorry, it's my mistake. I couldn't find the file defining MLTensor. Thanks.